crates/server/openapi.yaml

```
@@ -825,6 +822,44 @@ components:
           description: The algorithm-prefixed bytes of the signature (base64 encoded).
           pattern: ^[a-z0-9-]+:(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{4}|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{2}={2})$
           example: "ecdsa-p256:MEUCIQCzWZBW6ux9LecP66Y+hjmLZTP/hZVz7puzlPTXcRT2wwIgQZO7nxP0nugtw18MwHZ26ROFWcJmgCtKOguK031Y1D0="
+    MissingContentMap:
+      type: object
+      description: The map of content digest to missing content info.
+      patternProperties:
+        "^[a-z0-9-]+:[a-f0-9]+$":
+          "$ref": "#/components/schemas/MissingContent"
+      example:
+        ? "sha256:7d865e959b2466918c9863afca942d0fb89d7c9ac0c99bafc3749504ded9773"
+        : upload:
+            - type: http-post
```

We should keep to a JSON naming convention for enum values:

```suggestion
            - type: httpPost
```